src/Storages/ExternalStream/StorageExternalStream.h

```diff
@@ -60,6 +66,7 @@ class StorageExternalStream final : public shared_ptr_helper<StorageExternalStre
 
 private:
     std::unique_ptr<StorageExternalStreamImpl> external_stream;
+    std::shared_ptr<ExternalStreamCounter> external_stream_counter;
```

Comment:
since it is owned by StorageExternalStreamImpl, we don't need to declare it here.
Remove this line